modules/local/format_taxonomy_sintax.nf

```diff
@@ -0,0 +1,35 @@
+process FORMAT_TAXONOMY_SINTAX {
+    label 'process_low'
+
+    conda "conda-forge::sed=4.7"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img' :
+        'biocontainers/biocontainers:v1.2.0_cv1' }"
```

Comment:
I assume this should not work any more with template update 2.8 and instead a `docker.io/` as in https://github.com/nf-core/ampliseq/blob/8f0c71186fcf126d1c9c996bc91a2f88c1cd57eb/modules/local/format_taxonomy.nf#L7 would be required before.